Header header
uint16 x
uint16 y
string[] names
float64[] rel_probs
